src/models/IDE/Intellij.py

```diff
@@ -23,8 +24,14 @@ def __init__(self):
         self.configurations = []
 
     def get_ide_version(self, ide_directory):
-        pattern = re.compile(r"\d{3}\.\d{4}\.\d{2}")
-        return [name for name in os.listdir(ide_directory) if pattern.search(name) is not None][0]
+        info_file_path = os.path.join(ide_directory, 'product-info.json')
+        try:
+            with open(info_file_path, 'r') as info_file:
```

Comment:
You can use the `read_file` function in the util folder and parse the content

Reply:
done